packages/react-charting/src/utilities/gradients.ts

```diff
@@ -0,0 +1,146 @@
+export const DataVizGradientPalette = {
+  gradient1: 'default.1',
+  gradient2: 'default.2',
+  gradient3: 'default.3',
+  gradient4: 'default.4',
+  gradient5: 'default.5',
+  gradient6: 'default.6',
+  gradient7: 'default.7',
+  gradient8: 'default.8',
+  gradient9: 'default.9',
+  gradient10: 'default.10',
+  // extended gradients
+  /* gradient1Ext: 'extended.1',
```

Comment:
why are extended gradients commented out

Reply:
the extended gradients haven't been defined as yet. Will include them shortly.

Reply:
Added